runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/graph/LengthPrefixUnknownCoders.java

```diff
@@ -74,6 +76,16 @@
           "kind:void");
 
   private static final String LENGTH_PREFIX_CODER_TYPE = "kind:length_prefix";
+  private static Set<Node> sdkToRunnerBoundaries = new HashSet<>();
```

Comment:
You can't use statics because LengthPrefixUnknownCoders could be called from multiple threads.